Makefile

```
@@ -41,6 +49,7 @@ test:  sdk-test
 lint:  sdk-lint
 cover: sdk-cover
 tidy:  sdk-tidy
+bench: sdk-codec-bench
```

checkmake (0.3.2)

[warning] Target "bench" should be declared PHONY.

(phonydeclared)